decrepit/xts/xts.c

```diff
@@ -193,10 +215,32 @@ static int aes_xts_cipher(EVP_CIPHER_CTX *ctx, uint8_t *out,
       !xctx->xts.key2 ||
       !out ||
       !in ||
-      len < AES_BLOCK_SIZE ||
-      !CRYPTO_xts128_encrypt(&xctx->xts, ctx->iv, in, out, len, ctx->encrypt)) {
+      len < AES_BLOCK_SIZE) {
```

Comment:
Haven't studied this scheme in depth. I see the underlying implementation has this limitation. Is it inherent to the XTS scheme or just an implementation choice?

Reply:
From IEEE 1619-2007:
> 4.3.1 Data unit: Within IEEE Std 1619, 128 or more bits of data within a key scope. 
> Sec 5.1: The data unit size shall be at least 128 bits.